src/modules/workspace/tabs/AgentsTab.tsx

```
@@ -13,14 +13,15 @@
 import { useTranslation } from "react-i18next";
-import { WorkspaceData, Agent, LLMOption } from "../../types/Types";
+import { WorkspaceData, Agent, LLMOption } from "../types/Types";
 import { X, Plus, Trash2, Edit } from "lucide-react";
 
 interface AgentsTabProps {
   workspaceData: WorkspaceData;
+  onTabChanges?: () => void;
 }
 
-const AgentsTab: React.FC<AgentsTabProps> = ({ workspaceData: workspaceData }) => {
+const AgentsTab: React.FC<AgentsTabProps> = ({ workspaceData: workspaceData, onTabChanges }) => {
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)